### DISCUSSION TITLE
Search options including tags

### DISCUSSION THREAD

**Discussion by Vantropix** [ppy/osu#17476] [Ideas] [answered]
Since the options menu might either get crowded or already feel crowded depending on who you ask i propose a way to improve the search function of it.

Search is great on it's own but what if settings also came with a tag,
Let's take "Hold-To-Confirm activation time" as an example, it might be worth giving it general tags that would relate to what the setting changes, so in this case one thing that changes with this is pausing so a 'pause' and 'menu' tag might be applicable.
It's up to interpretation what tags need to be on what settings but it would majorly help in the settings for newbies and others who don't know settings by name and/or can't find the appropriate setting for a thing that exists but they can't find.

**Comment by bdach** [comment, marked as answer]
this is already how the search works. the tags are not visible anywhere, but it does work like that.
